table/tables/tables_test.go

```diff
@@ -622,79 +614,85 @@ func (ts *testSuite) TestHiddenColumn(c *C) {
 		"f|tinyint(4)|YES||<nil>|VIRTUAL GENERATED"))
 }
 
-func (ts *testSuite) TestAddRecordWithCtx(c *C) {
-	_, err := ts.se.Execute(context.Background(), "DROP TABLE IF EXISTS test.tRecord")
-	c.Assert(err, IsNil)
-	_, err = ts.se.Execute(context.Background(), "CREATE TABLE test.tRecord (a bigint unsigned primary key, b varchar(255))")
-	c.Assert(err, IsNil)
-	tb, err := ts.dom.InfoSchema().TableByName(model.NewCIStr("test"), model.NewCIStr("tRecord"))
-	c.Assert(err, IsNil)
+func TestAddRecordWithCtx(t *testing.T) {
+	t.Parallel()
+	store, dom, clean := testkit.CreateMockStoreAndDomain(t)
+	defer clean()
+	tk := testkit.NewTestKit(t, store)
+	_, err := tk.Session().Execute(context.Background(), "DROP TABLE IF EXISTS test.tRecord")
+	require.NoError(t, err)
+	_, err = tk.Session().Execute(context.Background(), "CREATE TABLE test.tRecord (a bigint unsigned primary key, b varchar(255))")
+	require.NoError(t, err)
+	tb, err := dom.InfoSchema().TableByName(model.NewCIStr("test"), model.NewCIStr("tRecord"))
+	require.NoError(t, err)
 	defer func() {
-		_, err := ts.se.Execute(context.Background(), "DROP TABLE test.tRecord")
-		c.Assert(err, IsNil)
+		_, err := tk.Session().Execute(context.Background(), "DROP TABLE test.tRecord")
+		require.NoError(t, err)
 	}()
```

Comment:
One thing I noticed is that we may be able to remove `DROP TABLE`, then cleanup function. Because every test is using a different mock store. You could investigate about that.

Reply:
create table -> do test -> drop table, it feels nature and more like an  integration tests.
it seems duplicated, but it also make sure `drop table` works and `clean up`  works.

Reply:
what's your idea?
if you insist on remove is better, i can do that job

Reply:
I won't insist anything as long as it is acceptable.